### PR TITLE
More FPS chart cleanup.

### PR DESCRIPTION
**Pull request by terrylucas**
- Fixed hover to show first data points at x coord -1.
- Hide xaxis ticks and labels while using rangeslider.
- Always show bar charts with consistent width.

### CODE REVIEW

File: packages/devtools/lib/src/timeline/frames_bar_chart.dart [hunk 1]
```diff
@@ -101,6 +103,23 @@ class PlotlyDivGraph extends CoreElement {
     }
   }
 
+  void _plotlyHover(DataEvent data) {
+    final List<HoverFX> hoverDisplay = [];
+
+    for (Point pt in data.points) {
+      final int ptNumber = pt.pointNumber;
+      final int x = pt.data.x[ptNumber];
+      // Only display the hover if its not the first data pointfor each trace
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/frames_bar_chart.dart]: nit:
pointfor --> point for

File: packages/devtools/lib/src/timeline/frames_bar_plotly.dart [hunk 2]
```diff
@@ -25,12 +32,10 @@ class FramesBarPlotly {
...
-        ticks: '',
-        showgrid: false,
-        showticklabels: false,
-        rangeslider: RangeSlider(
-          rangemode: 'nonnegative',
-          autorange: true,
+        rangeslider: RangeSlider(),
+        // Hide ticks by using font color of bgColor.
+        tickfont: Font(
+          color: 'white',
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/frames_bar_plotly.dart]: Add TODO to make this color work with the dark theme.

File: packages/devtools/lib/src/timeline/frames_bar_chart.dart (hunk 1 above)
Comment (terrylucas) [reply packages/devtools/lib/src/timeline/frames_bar_chart.dart]: Done

**Comment by jacob314** [comment]
lgtm! Saw a live demo and this makes the chart look much better when drawing the first few frames.

File: packages/devtools/lib/src/timeline/frames_bar_plotly.dart (hunk 2 above)
Comment (terrylucas) [reply packages/devtools/lib/src/timeline/frames_bar_plotly.dart]: Done.

**Pull request merged**